src/modules/commander/state_machine_helper.h

```diff
@@ -58,6 +58,16 @@ typedef enum {
 
 } transition_result_t;
 
+enum class link_loss_actions_t
+{
+	DISABLED = 0,
+	AUTO_LOITER = 1,
+	AUTO_RTL = 2,
+	AUTO_LAND = 3,
+	AUTO_RECOVER = 4,
+	TERMINATE = 5,
+	LOCKDOWN = 6,
```

Comment:
Currently flight termination is implemented differently for fixed wing. For MC it will be pretty much the same. But in the future Terminate can for example open parachute, or trigger some other flight termination procedure, while lockdown is always just motors lockdown